templates/zerver/help/keyboard-shortcuts.md

```diff
@@ -35,6 +35,8 @@ below, and add more to your repertoire as needed.
 
 * **Next unread topic**: `n`
 
+* **Next unread private message**: `p`
```

Comment:
Do we want these to say "next unread private message" or "next unread private message conversation"?

Reply:
We do say "Private Messages", so I feel "next unread private message" goes well.